src/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -593,13 +595,16 @@ class PackPageItem {
                 product        : shipmentItem?.product,
                 lotNumber      : shipmentItem?.lotNumber,
                 expirationDate : shipmentItem?.expirationDate?.format("MM/dd/yyyy"),
-                binLocationName: shipmentItem?.binLocation?.name,
+                binLocation    : shipmentItem?.binLocation,
                 uom            : shipmentItem?.product?.unitOfMeasure,
                 quantityShipped: shipmentItem?.quantity,
                 recipient      : shipmentItem?.recipient,
                 palletName     : palletName,
                 boxName        : boxName,
                 sortOrder      : sortOrder,
+                // deprecated
+                "binLocation.id"  : shipmentItem?.binLocation?.id,
```

Comment:
I think it should be `binLocationName` instead of `"binLocation.id"` and `"binLocation.name"`

Reply:
@awalkowiak You're right. I've updated it.